server/Makefile

```diff
@@ -9,7 +9,7 @@ clean:
 .PHONY: unit-test
 unit-test:
 	PYTHONWARNINGS=ignore:ResourceWarning coverage run \
-		--source=app,cli,common,compute,converters,data_anndata,data_common,data_cxg,eb \
+		--source=app,cli,common,compute,cache,dataset,eb \
```

Comment:
Why was cache added here?

Reply:
can we also remove cli?

Reply:
`cache` being here is almost certainly a merge error. 

TODO: remove these

Reply:
removed